.docker/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# start with official ruby docker image as base
+FROM ruby:3.1.2
+
+# set working directory within container
+WORKDIR /usr/src/app
+
+# pull in ruby (jekyll) and python (cite process) package info
+COPY Gemfile Gemfile.lock ./_cite/requirements.txt .
```

Comment:
This line produces a linting error in VS Code with the Docker extension installed:
"When using COPY with more than one source file, the destination must be a directory and end with a / or a \".

The fix would be to stick a "/" after the ".".

Reply:
👍